app/src/main/java/org/oppia/android/app/profileprogress/ProfileProgressHeaderViewModel.kt

```diff
@@ -27,6 +25,14 @@ class ProfileProgressHeaderViewModel(
 
   private var recentlyPlayedTopicCount: Int = 0
 
+  private fun getRecentlyPlayedActivityTitle(): RecentlyPlayedActivityTitle {
+    return if (completedStoryCount.get()!! >= 2) {
```

Comment:
We should be defaulting in the case where completedStoryCount hasn't been defined yet, rather than throwing a NPE and crashing the app.

Reply:
Handled NPE